_src/index.ts

```diff
@@ -8,3 +7,4 @@ import './notification.ts';
 import './subscribe.ts';
 import './scroll.ts';
 import './deploy-to-github.ts';
+import './doc-utils';
```

Comment:
Do we need t load this code on every page of the documentation? Is it worth the bundle size increase (and performance decrease), or should we only load it on specific pages?

Reply:
@StephenFluin resolved